File: examples/hello.ts
// Ejemplo mínimo para outline / LSP typescript-ls en tgdb.
// (sin DAP TypeScript aún)
// Compilar/ejecutar: npx ts-node hello.ts  o  tsc hello.ts && node hello.js

function greet(name: string): void {
  console.log(`hello, ${name}`);
}

greet("tide");

File: examples/hello.rs
//! Ejemplo mínimo para outline / LSP rust-analyzer en tgdb.
//! Requiere el `Cargo.toml` de este directorio (`examples/`).
//! Compilar: `cargo build --manifest-path examples/Cargo.toml --bin hello`
//! O: `rustc -g hello.rs`

fn greet(name: &str) {
    println!("hello, {name}");
}

fn main() {
    greet("tide");
   
    
}